VideoLocker/res/values/styles.xml

```diff
@@ -222,6 +222,21 @@
         <item name="android:maxHeight">3dip</item>
     </style>
 
+    <style name="edX.Widget.Switch" parent="Widget.AppCompat.CompoundButton.Switch">
+        <item name="android:switchTextAppearance">@style/default_switch</item>
+        <item name="android:showText">true</item>
+        <item name="android:textOff">@string/toggle_turn_off</item>
+        <item name="android:textOn">@string/toggle_turn_on</item>
+        <item name="android:thumb">@drawable/switch_inner</item>
+        <item name="android:thumbTextPadding">8dp</item>
+        <item name="android:track">@drawable/switch_track</item>
+        <item name="android:layout_width">90dp</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_gravity">center</item>
+        <item name="android:switchMinWidth">80dp</item>
+        <item name="android:textAllCaps">true</item>
+    </style>
+
     <style name="default_switch" parent="@style/semibold_text">
         <item name="android:textColor">@color/switch_text_color</item>
```

Comment:
In the switch_text_color.xml file:
1. `@color/switch_disable_text_color` can be replaced with `@color/edx_grayscale_neutral_base` making the color definition `switch_disable_text_color` useless and removable.
2. `@color/black` can be replaced with `@color/edx_grayscale_neutral_black_t`.